opam

```diff
@@ -20,7 +24,7 @@ depends: [
   "rpc"
   "fd-send-recv"
   "uuidm"
-  "xapi-stdext" {>= "2.0.0"}
```

Comment:
Is this version constraint unnecessary/incorrect? I think it might be good to have these as many of these as possible to ensure that we have a consistent set of packages in `xs-opam` and prevent compilation errors.